FlowCrypt/src/main/java/com/flowcrypt/email/security/pgp/PgpEncrypt.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * © 2016-present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com
+ * Contributors: DenBond7
+ */
+
+package com.flowcrypt.email.security.pgp
+
+import org.bouncycastle.bcpg.ArmoredInputStream
+import org.bouncycastle.openpgp.PGPPublicKeyRingCollection
+import org.pgpainless.PGPainless
+import java.io.ByteArrayInputStream
+import java.io.IOException
+import java.io.InputStream
+import java.io.OutputStream
+
+/**
+ * @author Denis Bondarenko
+ *         Date: 3/16/21
+ *         Time: 3:28 PM
+ *         E-mail: DenBond7@gmail.com
+ */
+object PgpEncrypt {
+  @Throws(IOException::class)
+  fun encryptFile(srcInputStream: InputStream, destOutputStream: OutputStream, pgpPublicKeyRingCollection: PGPPublicKeyRingCollection) {
+    srcInputStream.use { srcStream ->
+      destOutputStream.use { outStream ->
+        val builder = PGPainless.encryptAndOrSign()
+            .onOutputStream(outStream)
+            .toRecipients(pgpPublicKeyRingCollection)
+            .usingSecureAlgorithms()
+
+        builder.doNotSign().noArmor().use { encryptionStream ->
```

Comment:
@tomholub Please pay attention to this line

Reply:
Looks correct. We don't support signing on Android yet, and encrypted files should not be armored to prevent size bloat and inefficiency.